erpnext/accounts/doctype/bulk_statement_of_accounts/bulk_statement_of_accounts.json

```diff
@@ -0,0 +1,284 @@
+{
+ "actions": [],
+ "allow_workflow": 1,
+ "autoname": "Prompt",
+ "creation": "2020-05-22 16:46:18.712954",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "section_break_11",
+  "from_date",
+  "company",
+  "account",
+  "group_by",
+  "cost_center",
+  "column_break_14",
+  "to_date",
+  "finance_book",
+  "currency",
+  "project",
+  "preferences",
+  "include_ageing",
+  "ageing_based_on",
+  "section_break_14",
+  "orientation",
+  "section_break_1",
```

Comment:
Move "Print Preferences" and "Auto Email" section to the end, after the customers section. The reason is GL Filters and Customers are similar fields.